Microsoft.DotNet.Interactive/FormattedValue.cs

```diff
@@ -28,22 +29,30 @@ public static IReadOnlyCollection<FormattedValue> FromObject(object value)
         {
             var type = value?.GetType();
 
-            var mimeType = MimeTypeFor(type);
+            var mimeTypes = MimeTypesFor(type).Distinct();
```

Comment:
It seems like the mime types uniqueness should be enforced at the registration level.

Reply:
gone